f2/src/forms/ContactInfoForm.js

```diff
@@ -39,9 +39,15 @@ export const ContactInfoForm = (props) => {
       ) : null}
       <Formik
         initialValues={contactInfo}
-        validationSchema={validationSchema}
-        onSubmit={(values) => {
-          props.onSubmit(values)
+        onSubmit={async (values, { setErrors }) => {
+          const errors = !fyiForm
+            ? onSubmitValidation(values)
```

Comment:
Why manually set errors here? Shouldn't we just set the custom validation like: `validation={validationFunction}`?

Reply:
Replaced with new code submitted